backend/spec/factories/groups.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :group do
+    author_last_name { 'SAWQ' }
```

Comment:
Make author last name valid (not Upper case)

Reply:
> Make author last name valid (not Upper case)

It's not make sense